### PR TITLE
Added assert_properties

### PR DESCRIPTION
**Pull request by StephMc**

### CODE REVIEW

File: test/extra-asserts.js [hunk 1]
```diff
@@ -245,7 +254,80 @@ function autoTestRunner(){
...
+// Takes in a testRecord
 function assert_properties(test){
+  var object = test.object;
+  var targets = test.targets;
+  var message = test.message;
+  var epsilon = test.epsilon;
+  // Displays the actual current time, not just the time the test
+  // thinks it is (from test.time)
+  var time = parentAnimation.iterationTime === null ?
+      0 : parentAnimation.iterationTime;
```
Comment (DmitryBaranovskiy) [inline test/extra-asserts.js]: `var time = Number(parentAnimation.iterationTime);`

File: test/extra-asserts.js [hunk 2]
```diff
@@ -245,7 +254,80 @@ function autoTestRunner(){
...
+
+  // SVGs have to be handled differently to divs
+  var isSVG = (object.nodeName != "DIV");
+
+  // Create an element of the same type as testing so the style can be applied
+  // from the test. This is so the css property (not the -webkit-does-something
+  // tag) can be read.
+  var tempOb = document.createElement(object.nodeName);
```
Comment (DmitryBaranovskiy) [inline test/extra-asserts.js]: Shouldn’t it be `createElementNS("http://www.w3.org/2000/svg", object.nodeName);` for SVG?

**Comment by StephMc** [comment]
I've updated the code with the suggestions

**Comment by StephMc** [comment]
I've updated the code with the suggestions

**Comment by StephMc** [comment]
I've updated the code with the suggestions

**Pull request merged**